SU2 - CCX test fail building

Using latest `master` at 4d9603bbbe74424e031b4388d7e3de16c5be4bc9

Docker 18.09.3, Linux 5.0.4.

```
mpicc -Wall -O3 -fopenmp -I./ -I./adapter -I/CalculiX/ccx_2.15/src -I/spooles.2.2 -I/precice/src -I/usr/local/ARPACK  -DARCH="Linux" -DSPOOLES -DARPACK -DMATRIXSTORAGE -c adapter/PreciceInterface.c -o bin/PreciceInterface.o
adapter/PreciceInterface.c:13:38: fatal error: precice/SolverInterfaceC.h: No such file or directory
compilation terminated.
Makefile:65: recipe for target 'bin/PreciceInterface.o' failed
make: *** [bin/PreciceInterface.o] Error 1
The command '/bin/sh -c make' returned a non-zero code: 2
```
Complete console output: [Output.txt](https://github.com/precice/systemtests/files/3004385/Output.txt)

--- a/Test_of-ccx.Ubuntu1604/Makefile_of_ccx
+++ b/Test_of-ccx.Ubuntu1604/Makefile_of_ccx
@@ -3,6 +3,8 @@ CCX			= /CalculiX/ccx_2.15/src
 SPOOLES			= /spooles.2.2
 ARPACK			= /usr/local/ARPACK
 PRECICE_ROOT		= /precice
+PKGCONF_CFLAGS  = $(shell pkg-config --cflags libprecice)
+PKGCONF_LIBS    = $(shell pkg-config --libs libprecice)
 
 # Specify where to store the generated .o files
 OBJDIR 		= bin
@@ -13,27 +15,16 @@ INCLUDES = \
 	-I./adapter \
 	-I$(CCX) \
 	-I$(SPOOLES) \
-	-I$(PRECICE_ROOT)/src \
+	$(PKGCONF_CFLAGS) \
     	-I$(ARPACK) \
 
 LIBS = \
 	$(SPOOLES)/spooles.a \
 	$(ARPACK)/libarpack_INTEL.a \
-    	-lm -lc -lpthread -llapack -lblas \
-    	-L$(PRECICE_ROOT)/build/last \
-    	-lprecice \
-    	-lboost_log \
-    	-lboost_log_setup \
-    	-lboost_thread \
-    	-lboost_system \
-    	-lboost_filesystem \
-	-lboost_program_options \
-    	-lpython2.7 \
-    	-lstdc++ \
-	-L$(PETSC_DIR)/$(PETSC_ARCH)/lib -lpetsc \
-    	-lmpi_cxx \
-    	-lyaml-cpp \
-        -lxml2
+    	-lm -lc -lpthread\
+			$(PKGCONF_LIBS) \
+			-lstdc++ \
+			-lyaml-cpp \
 
 
 # Compilers and flags

--- a/Test_of-ccx.Ubuntu1604/Makefile_of_ccx
+++ b/Test_of-ccx.Ubuntu1604/Makefile_of_ccx
@@ -22,9 +22,9 @@ LIBS = \
 	$(SPOOLES)/spooles.a \
 	$(ARPACK)/libarpack_INTEL.a \
     	-lm -lc -lpthread\
-			$(PKGCONF_LIBS) \
-			-lstdc++ \
-			-lyaml-cpp \
+      $(PKGCONF_LIBS) \
+      -lstdc++ \
+      -lyaml-cpp \
 
 
 # Compilers and flags

--- a/Test_su2-ccx.Ubuntu1604/Makefile_su2-ccx
+++ b/Test_su2-ccx.Ubuntu1604/Makefile_su2-ccx
@@ -3,6 +3,8 @@ CCX			= /CalculiX/ccx_2.15/src
 SPOOLES			= /spooles.2.2
 ARPACK			= /usr/local/ARPACK
 PRECICE_ROOT		= /precice
+PKGCONF_CFLAGS  = $(shell pkg-config --cflags libprecice)
+PKGCONF_LIBS    = $(shell pkg-config --libs libprecice)
 
 # Specify where to store the generated .o files
 OBJDIR 		= bin
@@ -13,27 +15,17 @@ INCLUDES = \
 	-I./adapter \
 	-I$(CCX) \
 	-I$(SPOOLES) \
-	-I$(PRECICE_ROOT)/src \
+	$(PKGCONF_CFLAGS) \
+    	-I$(ARPACK) \
     	-I$(ARPACK) \
 
 LIBS = \
 	$(SPOOLES)/spooles.a \
 	$(ARPACK)/libarpack_INTEL.a \
-    	-lm -lc -lpthread -llapack -lblas \
-    	-L$(PRECICE_ROOT)/build/last \
-    	-lprecice \
-    	-lboost_log \
-    	-lboost_log_setup \
-    	-lboost_thread \
-    	-lboost_system \
-    	-lboost_filesystem \
-	-lboost_program_options \
-    	-lpython2.7 \
-    	-lstdc++ \
-	-lpetsc \
-    	-lmpi_cxx \
-    	-lyaml-cpp \
-        -lxml2
+    	-lm -lc -lpthread\
+			$(PKGCONF_LIBS) \
+			-lstdc++ \
+			-lyaml-cpp \
 
 
 # Compilers and flags

--- a/Test_su2-ccx.Ubuntu1604/Makefile_su2-ccx
+++ b/Test_su2-ccx.Ubuntu1604/Makefile_su2-ccx
@@ -17,15 +17,14 @@ INCLUDES = \
 	-I$(SPOOLES) \
 	$(PKGCONF_CFLAGS) \
     	-I$(ARPACK) \
-    	-I$(ARPACK) \
 
 LIBS = \
 	$(SPOOLES)/spooles.a \
 	$(ARPACK)/libarpack_INTEL.a \
     	-lm -lc -lpthread\
-			$(PKGCONF_LIBS) \
-			-lstdc++ \
-			-lyaml-cpp \
+      $(PKGCONF_LIBS) \
+      -lstdc++ \
+      -lyaml-cpp \
 
 
 # Compilers and flags

